README.md

```diff
@@ -26,17 +26,18 @@ Deploy and log in with username *admin* and the password that's set up during de
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.2</b> or higher
+ * NodeJS 8.0 or higher
  * [Composer](https://getcomposer.org/)
+ * NPM
+ * make
```

Comment:
If we list `make` here we should also list all `apt` dependencies for LORIS, as detailed in #4365:

                               "curl",
                               "wget",
                               "zip",
                               "unzip",
                               "php-json",
                               "make",
                               "software-properties-common",
                               "php-ast",
                               "php$required_php",
                               "php$required_php-mysql",
                               "php$required_php-xml",
                               "php$required_php-json",
                               "php$required_php-mbstring",
                               "php$required_php-gd",
                               "libapache2-mod-php$required_php",

`$required_php` should be replaced with PHP 7.2 for now.

Alternatively, I think a better solution would be to merged that PR so that all of these steps are done automatically for the users.